data/translated-rules-en-us.yaml

```diff
@@ -17010,21 +17010,21 @@ transport . avion:
   description: |
     ![](https://images.unsplash.com/photo-1525624286412-4099c83c1bc8?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80)
 
-    The airplane's footprint is substantial. It is not necessarily more
-    polluting *per kilometer* than a car that moves only one person, but the
-    distance and especially the distance covered in one hour (commonly
-    called speed), allows to change continent for a weekend.
-
-    Let's also keep in mind that the average French person does not fly or
-    does not fly much. Most of the aviation footprint is therefore
-    concentrated on 10% or less of the French population.
+    The footprint of the plane is substantial. It is not necessarily more polluting *per kilometer* than a car that only moves one person,
+    but the distance and especially the distance covered in one hour (commonly called speed), allows to change continent for a weekend.
+
+    Let's also keep in mind that the average French person does not fly or does not fly much. Most of the aviation footprint is therefore concentrated on 10% or less of the French population.
+
+    > For the definition of flight ranges, from short-haul to long-haul, we use <a href="https://bilans-ges.ademe.fr/forum/viewtopic.php?t=4192#p5530">this discussion</a> from the Base Carbone forum.
   description.lock: |
     ![](https://images.unsplash.com/photo-1525624286412-4099c83c1bc8?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80)
 
     L'empreinte de l'avion est conséquente. Il n'est pas forcément plus polluant *au kilomètre* qu'une voiture qui ne déplace qu'une personne,
     mais la distance et surtout la distance parcourue en une heure (qu'on appelle couramment la vitesse), permet de changer de continent pour un week-end.
 
     Gardons aussi en tête que le Français moyen ne prend pas ou peu l'avion. L'essentiel de l'empreinte de l'aviation est donc concentrée sur 10% ou moins des Français.
+
+    > Pour la définition des gammes de vol, de court-courrier à long-courrier, nous utilisons [cette discussion](https://bilans-ges.ademe.fr/forum/viewtopic.php?t=4192#p5530) du forum Base Carbone.
   note: |
     We have chosen to ask for the total number of flight hours, rather than
     asking the user to fill in each of his flights via the departure and
```

Comment:
il n'y a rien après le and, c'est normal ?

Reply:
Si mais à mon avis c'est que dans les "diffs" tu n'avais pas le fichier complet, il y a bien la suite

Dans la partie "files changed", seules les modifs sont mises en valeur, sans doute que cette partie n'a pas été modifiée dans cette PR donc on ne voit pas la suite